p2p/src/sync/tests/A_REMOVE_block_response.rs

```diff
@@ -0,0 +1,305 @@
+// Copyright (c) 2022 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
This file has an interesting filename... what's the story on this?

Reply:
It is a story about an oversight. 🙃  While writing new tests I preserved the old ones to double check that I wasn't missing some of the test cases, but I forgot to remove this file after that.